plugins/sampledata/multilang/multilang.php

```diff
@@ -1143,7 +1143,7 @@ private function addArticle($itemLanguage, $categoryId)
 
 		$query = $db->getQuery(true)
 			->insert($db->quoteName('#__content_frontpage'))
-			->values($newId . ', 0');
+			->values($newId . ', 0, null, null');
```

Comment:
```suggestion
			->values($newId . ', 0, NULL, NULL');
```
Is SQL NULL, not PHP null, so it should be uppercase.